lib/sync/worker.js

```diff
@@ -1,6 +1,27 @@
 var syncUtil = require('./util');
 var debug = syncUtil.debug;
 var metrics = require('./sync-metrics');
+var backoff = require('backoff');
+
+function initBackoff(backOffOpts) {
+  var bo;
+  if (backOffOpts.strategy === 'exp') {
+    bo = new backoff.ExponentialStrategy({initialDelay: backOffOpts.min, maxDelay: backOffOpts.max});
+  } else if (backOffOpts.strategy === 'fib') {
+    bo = new backoff.FibonacciStrategy({initialDelay: backOffOpts.min, maxDelay: backOffOpts.max});
+  } else {
+    //if no valid value is given, just return the default minimum delay. Can be used to turn off backoff.
```

Comment:
I think it should ignore the case. I don't think thrown an exception here is right as it doesn't really affect the running of the apps even there is no backoff. I think we can print out a warning message here to make it obvious that the default strategy is being used.

Given that there is only 3 charactors, I think it's hard to mis-spell.